src/components/chat/chatContainer.jsx

```diff
@@ -1,22 +1,13 @@
 import React, { Component } from 'react';
 
-export default class ChatContainer extends Component {
-  constructor(props) {
-    super(props);
-    this.sendMessage = props.sendMessage;
-    this.handleChangeAuthor = props.handleChangeAuthor;
-    this.handleChangeText = props.handleChangeText;
-  }
-
-  render() {
-    return (
-      <form>
-        <div className='center'>
-          <button className='btn btn-primary' onClick={this.sendMessage}>Send Message</button>
-          <input onChange={this.handleChangeAuthor} placeholder='Name'></input>
-          <input onChange={this.handleChangeText} placeholder='Message'></input>
-        </div>
-      </form>
-    )
-  }
+export default function ChatContainer({ handleChangeAuthor, handleChangeText, sendMessage }) {
```

Comment:
when you have a component that is purely for display, consider changing to a pure functional component rather than a class based component. the function when used as a react component will then receive an object as the argument. this object is being destructed here to extract out the necessary props.